com.unity.ml-agents.extensions/Runtime/QTTransform.cs

```diff
@@ -0,0 +1,96 @@
+using UnityEngine;
+
+namespace Unity.MLAgents.Extensions
+{
+    /// <summary>
+    /// Simple linear transform representation consisting of a rotation and translation.
+    /// When deriving the math, it can be helpful to think of the transform as a 4x4 block matrix:
+    /// <example>
+    /// | R | t |
+    /// ----+----
+    /// | 0 | 1 |
+    /// </example>
+    /// where R is a 3x3 rotation, t is a 3x1 translation, 0 is a 1x3 vector of 0s.
+    /// </summary>
+    public struct QTTransform
```

Comment:
Should some of these classes/struct be internal instead of public ?

Reply:
This one is removed, so it's moot.

I think at a minimum, the sensor and sensorcomponent classes should be public. We can debate about PoseExtractor and subclasses, but I'd default to public if possible.

Since these are going in the extensions package, we don't have to be as strict as we do for the core package. If they migrate to there, we can revisit them.